snappy_wrappers/wrappers/varfish_annotator/environment.yaml

```diff
@@ -4,6 +4,9 @@ channels:
 dependencies:
   - bcftools >=1.9
   - htslib >=1.9
+  - samtools >=1.9
   - varfish-annotator-cli ==0.27
   - openjdk >=8,<9
   - jq
+  - vcfpy
+  - pysam
```

Comment:
Do you need `pysam`? It seems like you are using a `subprocess` call.

Reply:
vcfpy needs pysam as this is not int he bioconda package dependency